templates/blog/index.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<div class="card mb-4">
+    <div class="card-body">
+        <h1 class="card-title">Blog Posts</h1>
+        {% for post in post_list %}
+        <div class="card mb-3">
+            <div class="card-body">
+                <h5 class="card-title">{{ post.title }}</h5>
+                <p class="card-subtitle text-muted mb-2">{{ post.owner }} at {{ post.created_time }}</p>
+                <p class="card-text">{{ post.content|slice:":200" }}...</p>
+                <a href="{% url 'blog:post-detail' post.id %}" class="btn btn-primary">Read More</a>
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.